swarm-test/src/lib.rs

```diff
@@ -200,26 +207,58 @@ where
 {
     type NB = B;
 
-    fn new_ephemeral(behaviour_fn: impl FnOnce(Keypair) -> Self::NB) -> Self
+    #[cfg(feature = "async-std")]
+    fn new_ephemeral(behaviour_fn: impl FnOnce(libp2p_identity::Keypair) -> Self::NB) -> Self
     where
         Self: Sized,
     {
+        use libp2p_core::{transport::MemoryTransport, upgrade::Version, Transport as _};
+        use libp2p_identity::Keypair;
```

Comment:
same as the question above, why is the `Keypair` import moved from the top of the file to duplicated here and in the `tokio` case?

Reply:
When both `async-std` and `tokio` runtimes are disabled, topilevel import of `Keypair` is unused and leads to clippy warnings

Reply:
ok, thanks for elaborating!